docs/sources/administration/configuration.md

```diff
@@ -909,7 +909,11 @@ Default is `false`.
 
 ### templates_pattern
 
-Default is `emails/*.html`.
+Enter a comma-separated list of template patterns. Default is `emails/*.html, emails/*.txt`.
+
+### content_types
+
+Enter a comma-separated list of content types that should be included in the emails that are sent. List the content types according descending preference. The order of the parts is significant as the mail clients will use the content type that is supported and most preferred by the sender. Supported content types are `text/html` and `text/plain`. Default is `text/html`.
```

Comment:
> List the content types according descending preference

I thought this meant that I should add the one with highest priority at the end. Might be me. Either rewrite it as "List the content types in the order of most preferred, e.g. `text/html,text/plain` for HTML as the most preferred" or either add an example similar to mine to make it easier to understand.

Reply:
Updated the docs and you were right about the `npm run build` indeed. Forgot to do that after resolving a merge conflict.